.github/workflows/centos8-wheel-tester.yaml

```diff
@@ -53,9 +53,11 @@ jobs:
               from cryptography.hazmat.backends.openssl.backend import backend
               print('Loaded: ' + backend.openssl_version_text())
               print('Linked Against: ' + backend._ffi.string(backend._lib.OPENSSL_VERSION_TEXT).decode('ascii'))
+          - PIP_NAME: sqlalchemy
```

Comment:
Can you place this in the AL2 and Ubuntu testers as well? Thanks.

Reply:
@geoffreyblake I have added the test for AL2 and Ubuntu testers as well. Please review the changes.